code/modules/supermatter/supermatter.dm

```diff
@@ -20,8 +20,8 @@
 #define DECAY_FACTOR 700			//Affects how fast the supermatter power decays
 #define CRITICAL_TEMPERATURE 5000	//K
 #define CHARGING_FACTOR 0.05
-#define DAMAGE_RATE_LIMIT 3			//damage rate cap at power = 300, scales linearly with power
-
+#define DAMAGE_RATE_LIMIT 2			//damage rate cap at power = 300, scales linearly with power
+#define SPACED_DAMAGE_FACTOR 0.1	//multiplier for damage taken in a vacuum, but on a tile. Used to prevent/configure near-instant explosions when vented
```

Comment:
so this reduces vacuum damage by 90% according to formula.

Reply:
because `number * 0.1 = 10% of the number`

Reply:
Yes. Added it as a configurable number in case anyone thinks it should be higher or lower.